src/routes/waitlist/+page.svelte

```diff
@@ -6,13 +6,13 @@
 </script>
 
 <svelte:head>
-    <title>Join the Waitlist | Lovat</title>
+    <title>Join our email list | Lovat</title>
```

Comment:
I prefer "mailing list"

We should also rename these pages and set up `301` redirects if it's transitioning from a waitlist to a mailing list.

I also think people might be more reluctant to sign up to recieve emails than they are to join a waitlist. We should be deliberate in what terminology we use for this.